coderd/insights_internal_test.go

```diff
@@ -148,3 +150,110 @@ func Test_parseInsightsStartAndEndTime(t *testing.T) {
 		})
 	}
 }
+
+func Test_parseInsightsInterval_week(t *testing.T) {
+	t.Parallel()
+
+	layout := insightsTimeLayout
+	now := time.Now().UTC()
+	y, m, d := now.Date()
+	today := time.Date(y, m, d, 0, 0, 0, 0, time.UTC)
+
+	thisHour := time.Date(y, m, d, now.Hour(), 0, 0, 0, time.UTC)
+	twoHoursAgo := thisHour.Add(-2 * time.Hour)
+	thirteenDaysAgo := today.AddDate(0, 0, -13)
+
+	sixDaysAgo := today.AddDate(0, 0, -6)
+	nineDaysAgo := today.AddDate(0, 0, -9)
+
+	type args struct {
+		startTime string
+		endTime   string
+	}
+	tests := []struct {
+		name         string
+		args         args
+		wantInterval codersdk.InsightsReportInterval
```

Comment:
This doesn't seem to be used and could be removed.

Reply:
Done